src/Integration/SLCore/SLCoreConstantsProvider.cs

```diff
@@ -33,9 +35,10 @@ namespace SonarLint.VisualStudio.Integration.SLCore
     public class SLCoreConstantsProvider : ISLCoreConstantsProvider
     {
         private readonly Lazy<string> ideName;
+        private readonly IVsVersion ideVersion;
 
         [ImportingConstructor]
-        public SLCoreConstantsProvider(IVsUIServiceOperation vsUiServiceOperation)
+        public SLCoreConstantsProvider(IVsUIServiceOperation vsUiServiceOperation, IVsVersionProvider vsVersionProvider)
         {
             // lazy is used to keep the mef-constructor free threaded and to avoid calling this multiple times
             ideName = new Lazy<string>(() =>
```

Comment:
Could this be moved to the IVsVersionProvider? Potentially with a rename. The Ide name and the Ide version seem to be related enough to be placed in the same class

Reply:
https://sonarsource.atlassian.net/browse/SLVS-1258